py/desispec/sky.py

```diff
@@ -243,7 +243,7 @@ def qa_skysub(param, frame, fibermap, skymodel):
     specmin, specmax = np.min(frame.fibers), np.max(frame.fibers)
     skyfibers=np.where((fibermap["OBJTYPE"]=="SKY")&
         (fibermap["FIBER"]>=specmin)&(fibermap["FIBER"]<=specmax))[0]
-    assert np.max(skyfibers) < 500
+    assert np.max(skyfibers) < 500 # only spectrograph 0??
```

Comment:
This is only insuring that there are fewer than 500 fibers.
These are indices, not the fiber values